### DISCUSSION TITLE
random interrupts to "verify my profile" are distractions and should never happen

### DISCUSSION THREAD

**Discussion by edw-eqix** [community/community#77249] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I recently clicked a github PR link and instead of being taken to the PR, I got forcibly redirected to a page that asked me to "verify" my profile information. When I clicked out, rather than ending up at my link, I landed on some random page (I don't recall exactly where). Getting hard-interrupted when I am busy trying to work is not good. I expect to be able to work at work. I do not accept that "verifying" my profile information, which has not, and does not change regularly, is ever a good use of my time. Even if it were, there are multiple places where github already forces interrupts that could be used. Please consider the vast, vast swathe of your users who have.. oh never mind.. you won't bother. Thanks for wasting my time.

**Comment by ghostinhershell** [comment]
Hey there!  👋

Thanks for posting in the GitHub Community and sharing your feedback, @edw-eqix. We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!

**Reply by edw-eqix** [reply]
This _is_ an accessibility issue, unless you're somehow not considering attention deficit disorders under that rubric.
